test/e2e/scenarios/scenario1.yaml

```diff
@@ -0,0 +1,40 @@
+id: "RG_CMD_basic_create_update"
```

Comment:
recommend naming the file `test/e2e/scenarios/replication_group/cmd_basic_create_update.yaml` or something like that instead of `scenario1.yaml` :)

Reply:
Updated file name.